### PR TITLE
[ADD] event_student_history

### PR DESCRIPTION
**Pull request by Daniel-CA**

### CODE REVIEW

**Comment by Daniel-CA** [comment]
@avanzosc/developers review please

**Review by alfredoavanzosc — APPROVED** [review]
:+1

File: event_student_history/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from . import models
```
Comment (oihane) [inline event_student_history/__init__.py]: falta el salto de linea
:wink:

**Comment by oihane** [comment]
FLAKE A REVISAR
```
event_student_history/models/event_student_history.py:4:1: F401 'odoo._' imported but unused
event_student_history/models/event_student_history.py:4:1: F401 'odoo.api' imported but unused
event_student_history/models/event_student_history.py:6:1: E302 expected 2 blank lines, found 1
event_student_history/models/event_student_history.py:8:1: W293 blank line contains whitespace
```

**Review by oihane — APPROVED** [review]

**Pull request merged**